app/src/main/res/layout/activity_book.xml

```diff
@@ -0,0 +1,161 @@
+                    android:layout_marginRight="@dimen/vote_star_margin"/>
+
+                <ImageView
+                    android:layout_width="@dimen/vote_star_size"
+                    android:layout_height="@dimen/vote_star_size"
+                    android:src="@drawable/ic_open_book"
+                    android:layout_marginRight="@dimen/vote_star_margin"/>
+
+                <ImageView
+                    android:layout_width="@dimen/vote_star_size"
+                    android:layout_height="@dimen/vote_star_size"
+                    android:src="@drawable/ic_open_book"
+                    android:layout_marginRight="@dimen/vote_star_margin"/>
+
+                <ImageView
+                    android:layout_width="@dimen/vote_star_size"
+                    android:layout_height="@dimen/vote_star_size"
+                    android:src="@drawable/ic_open_book"
+                    android:layout_marginRight="@dimen/vote_star_margin"/>
+
+                <ImageView
+                    android:layout_width="@dimen/vote_star_size"
+                    android:layout_height="@dimen/vote_star_size"
+                    android:src="@drawable/ic_open_book"
+                    android:layout_marginRight="@dimen/vote_star_margin"/>
+
+                <ImageView
+                    android:layout_width="@dimen/vote_star_size"
+                    android:layout_height="@dimen/vote_star_size"
+                    android:src="@drawable/ic_open_book"
+                    android:layout_marginRight="@dimen/vote_star_margin"/>
+
+                <ImageView
+                    android:layout_width="@dimen/vote_star_size"
+                    android:layout_height="@dimen/vote_star_size"
+                    android:src="@drawable/ic_open_book"
+                    android:layout_marginRight="@dimen/vote_star_margin"/>
+
+                <ImageView
+                    android:layout_width="@dimen/vote_star_size"
+                    android:layout_height="@dimen/vote_star_size"
+                    android:src="@drawable/ic_open_book"
+                    android:layout_marginRight="@dimen/vote_star_margin"/>
+
+                <ImageView
+                    android:layout_width="@dimen/vote_star_size"
+                    android:layout_height="@dimen/vote_star_size"
+                    android:src="@drawable/ic_close_book"/>
+            </LinearLayout>
+
+            <TextView
+                android:id="@+id/author_tv"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                tools:text="Agatha Christie"
+                android:textSize="@dimen/author_label_text_size"
+                android:textStyle="bold"/>
+
+            <TextView
+                android:id="@+id/category_tv"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                tools:text="Category"
```

Comment:
tutaj są teksty wpisane tak po prostu. Nie powinny! 
Musisz je usunąć 🗡  :D

Reply:
Bo to tylko *tools:text* :D

Reply:
Aaa.. my bad!